test/integration/features/user_creates_from_url.spec.js

```diff
@@ -0,0 +1,204 @@
+    createProjectPage.visit();
+    createProjectPage.createProject();
+    let catalogPage = new CatalogPage(fixturesProject);
+    // - add an image stream to that namespace
+    catalogPage.visit();
+    catalogPage.processImageStream(JSON.stringify(centosImageStream));
+    // - add a template to that namespace
+    catalogPage.visit();
+    catalogPage.saveTemplate(JSON.stringify(nodeMongoTemplate));
+  };
+
+  beforeAll(function() {
+    h.commonSetup();
+    h.login();
+    projectHelpers.deleteAllProjects();
+    setupEnv();
+  });
+
+  afterAll(function() {
+    projectHelpers.deleteAllProjects();
+    h.afterAllTeardown();
+  });
+
+  describe('create from URL', function() {
+
+    describe('using an image stream and image stream tag supplied as query string params', function() {
+
+      let name = 'nodejs-edited';
+      let sourceURI = 'https://github.com/openshift/nodejs-ex.git-edited';
+      let sourceRef = 'master-edited';
+      let contextDir = '/-edited';
+      let qs = '?imageStream=nodejs&imageTag=4&name=' + name + '&sourceURI=' + sourceURI + '&sourceRef=' + sourceRef + '&contextDir=' + contextDir + '&namespace=' + namespaceForFixtures;
+      let uri = 'project/' + project.name + 'create/fromimage' + qs;
+      let heading = 'Node.js 4';
+      let words = project.name.split(' ');
+      let timestamp = words[words.length - 1];
+
+      it('should display details about the the image', function() {
+        let createFromURLPage = new CreateFromURLPage();
+        createFromURLPage.visit(qs);
+        matchersHelpers.expectHeading(heading);
+      });
+
+      it('should load the image stream in to a newly created project', function(){
+        let createFromURLPage = new CreateFromURLPage();
+        createFromURLPage.visit(qs);
+        // NOTE: if the tabs are on the page because there are existing projects, click the 'Create New Project' tab first
+        createFromURLPage.clickCreateNewProjectTab();
+        projectHelpers.createProject(project, 'project/' + project['name'] + 'create/fromimage' + qs);
+        matchersHelpers.expectHeading(heading);
+        projectHelpers.deleteProject(project);
+      });
+
+      it('should load the image stream in to an existing project and verify the query string params are loaded in to the corresponding form fields', function(){
+        let createFromURLPage = new CreateFromURLPage();
+        projectHelpers.visitCreatePage();
+        projectHelpers.createProject(project);
+        createFromURLPage.visit(qs);
+        createFromURLPage.selectExistingProject(timestamp, uri);
+        matchersHelpers.expectHeading(heading);
+        let nameInput = element(by.model('name'));
+        expect(nameInput.getAttribute('value')).toEqual(name);
+        let sourceURIInput = element(by.model('buildConfig.sourceUrl'));
+        expect(sourceURIInput.getAttribute('value')).toEqual(sourceURI);
+        // click "Show advanced..." link to reveal hidden fields
```

Comment:
This could be pushed up into the page object as a method `.clickShowAdvanced()` & eliminate the comment.